data/strings/strings.txt

```diff
@@ -4036,6 +4036,99 @@
     zh-Hans = 书签上传失败。该文件可能已损坏或有缺陷。
     zh-Hant = 書籤上傳失敗。 檔案可能有毀損或是不完全。
 
+  [unknown_file_type]
+    comment = Failed to recognize the format of a bookmark file.
+    tags = android
+    en = The file type is not recognized by the app:\n%1$@
+    af = Die lêertipe word nie deur die toepassing herken nie:\n%1$@
+    ar = لم يتعرف التطبيق على نوع الملف:\n%1$@
+    az = Fayl növü tətbiq tərəfindən tanınmır:\n%1$@
+    be = Прыкладанне не распазнае тып файла:\n%1$@
+    bg = Типът на файла не се разпознава от приложението:\n%1$@
+    ca = L'aplicació no reconeix el tipus de fitxer:\n%1$@
+    cs = Typ souboru není aplikací rozpoznán:\n%1$@
+    da = Filtypen genkendes ikke af appen:\n%1$@
+    de = Der Dateityp wird von der App nicht erkannt:\n%1$@
+    el = Ο τύπος αρχείου δεν αναγνωρίζεται από την εφαρμογή:\n%1$@
+    es = La aplicación no reconoce el tipo de archivo:\n%1$@
+    et = Rakendus ei tunnista failitüüpi:\n%1$@
+    eu = Aplikazioak ez du fitxategi mota ezagutzen:\n%1$@
+    fa = نوع فایل توسط برنامه تشخیص داده نمی شود:\n%1$@
+    fi = Sovellus ei tunnista tiedostotyyppiä:\n%1$@
+    fr = Le type de fichier n'est pas reconnu par l'appli :\n%1$@
+    he = סוג הקובץ אינו מזוהה על ידי האפליקציה:\n%1$@
+    hi = फ़ाइल प्रकार ऐप द्वारा पहचाना नहीं गया है:\n%1$@
+    hu = Az alkalmazás nem ismeri fel a fájltípust:\n%1$@
+    id = Jenis file tidak dikenali oleh aplikasi:\n%1$@
+    it = Il tipo di file non è riconosciuto dall'applicazione:\n%1$@
+    ja = ファイルタイプがアプリに認識されない：\n%1$@
+    ko = 파일 형식이 앱에서 인식되지 않습니다:\n%1$@
+    lt = Programa neatpažįsta failo tipo:\n%1$@
+    mr = फाइल प्रकार ॲपद्वारे ओळखला जात नाही:\n%1$@
+    nb = Filtypen gjenkjennes ikke av appen:\n%1$@
+    nl = Het bestandstype wordt niet herkend door de app:\n%1$@
+    pl = Typ pliku nie jest rozpoznawany przez aplikację:\n%1$@
+    pt = O tipo de ficheiro não é reconhecido pela aplicação:\n%1$@
+    pt-BR = O tipo de arquivo não é reconhecido pelo aplicativo:\n%1$@
+    ro = Tipul de fișier nu este recunoscut de aplicație:\n%1$@
+    ru = Тип файла не поддерживается приложением:\n%1$@
+    sk = Aplikácia nerozpoznáva typ súboru:\n%1$@
+    sv = Filtypen känns inte igen av appen:\n%1$@
+    sw = Aina ya faili haitambuliwi na programu:\n%1$@
+    th = แอปไม่รู้จักประเภทไฟล์:\n%1$@
+    tr = Dosya türü uygulama tarafından tanınmıyor:\n%1$@
+    uk = Тип файлу не розпізнається програмою:\n%1$@
+    vi = Loại tệp không được ứng dụng nhận dạng:\n%1$@
+    zh-Hans = 应用程序无法识别文件类型：\n%1$@
+    zh-Hant = 應用程式無法識別該文件類型：\n%1$@
+
+  [failed_to_download_file]
+    comment = Failed to download a bookmark file.
```

Comment:
Since when downloading of bookmarks is supported in OM?

Reply:
Does it mean that OM can actually download bookmarks from remote URLs? Are there any proofs that its possible? AFAIU OM can open only local content URLs. Right?

Using the "Download" word now looks very misleading.